index.html

```diff
@@ -96,6 +100,145 @@ <h2>Software Developer</h2>
             </a>
         </div>    
         </section>
+<!--Recent work section-->
+        <section class="recent--work--container">
+            <div class="section--tittle">
+                <h2>MY RECENT <br/> WORK</h2>
+                <span class="devider">
+                    <img src="./assets/svg/Devider.svg" alt="Section devider">
+                </span>
+            </div> 
+               
+            <ul class="cards--container">
+
+                <li class="recent--work--card">
+                    <a href="#">
+                        <picture class="card--thumbnail">
+                          <img src="./assets/img/Snapshoot-Portfolio.png" alt="project screenshot">
```

Comment:
- [ ] **Your project images are broken. Please fix them**

![image](https://user-images.githubusercontent.com/59168430/120857815-7c406900-c59f-11eb-9c1d-4bd10214b432.png)

The project images are not loading because you have used the incorrect filename. Please fix the filename in HTML so that it matches the actual filename.